src/tangerine/agents/jira_agent.py

```diff
@@ -0,0 +1,72 @@
+import nltk
+import requests
+from langchain_core.prompts import ChatPromptTemplate
+from nltk.corpus import words
+
+import tangerine.config as cfg
+import tangerine.llm as llm
+
+# Ensure the corpus is downloaded
+nltk.download("words", quiet=True)
+
+# Load English words into a set for fast lookup
+english_words = set(words.words())
+
+
+class JiraAgent:
+    def __init__(self):
+        self.url = cfg.JIRA_AGENT_URL
+
+    def fetch(self, query: str):
+        user_list = self._find_usernames(query)
+        user_count = len(user_list)
+        users = ",".join(user_list)
+        query_url = f"{self.url}/?users={users}"
+        # Perform the GET request
+        response = requests.get(query_url, timeout=120)
+        # Check if the request was successful
+        if response.status_code == 200:
+            # Parse the JSON response
+            summaries = response.json()
+            # data is a list of strings. we need to convert it to a string
+            summaries = "\n".join(summaries)
+            clean_summaries = summaries.replace("<br>", "\n")
+            return_value = clean_summaries
+            if user_count > 1:
+                return_value = self._higher_order_summary(summaries)
+            return return_value
+        else:
+            # Handle the error
+            print(f"Error: {response.status_code}")
+            return "I tried getting info from Jira, but something went wrong."
```

Comment:
Should you wrap the `requests.get` in a try/except and return this string as well?